### PR TITLE
Add MatchRecorder (formerly ErikLogger)

### PR DESCRIPTION
**Pull request by ajnadel**
MatchRecorder will log data about the robot and allow us to piece together a tick-by-tick replay of what happened during a match.

### CODE REVIEW

File: src/org/usfirst/frc4904/robot/RobotMap.java [hunk 1]
```diff
@@ -100,6 +101,7 @@
 		public static CustomJoystick operatorStick;
 		public static TeensyController teensyStick;
 		public static PDP pdp;
+		public static PowerDistributionPanel CANPDPlogger;
```
Comment (carturn) [inline src/org/usfirst/frc4904/robot/RobotMap.java]: As soon as we have better brownouts finished, you will be able to get everything except temperature from the PDP. This is really, really preferable as PowerDistributionPanel throws unstoppable exceptions if the PDP is disconnected, which would kill robot code during a match.
Also, if you need temperature, I can add it too.

Comment (c0d3rman) [reply src/org/usfirst/frc4904/robot/RobotMap.java]: We don't need temperature

**Review by c0d3rman — APPROVED** [review]
Approved, though I'd like to switch to the new PDP before we merge.

**Comment by tylkahn** [comment]
Looks good to me.

**Pull request merged**